tests/functional/cli/admin/test_service_list.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2019 OpenIO SAS, as part of OpenIO SDS
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3.0 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.
+
+import tempfile
+from tests.utils import random_str
+from tests.functional.cli import CliTestCase
+
+
+class ServiceListTest(CliTestCase):
+
+    def test_meta2_list_containers(self):
+        opts = self.get_format_opts()
+        container = random_str(16)
+        output = self.openio('container create %s %s --oio-account %s'
+                             % (container, opts, self.account))
+        self.assertOutput('%s True' % container, output.strip())
+        opts = self.get_format_opts(fields=['meta2'])
+        output = self.openio('container locate %s %s  --oio-account %s'
+                             % (container, opts, self.account))
+        meta2s = output.strip().split(' ')
+        opts = self.get_format_opts()
+        fullname = '/'.join([self.ns, self.account, container])
+        for meta2 in meta2s:
+            if meta2.endswith(','):
+                meta2 = meta2[:-1]
+            output = self.openio_admin(
+                'meta2 list containers  %s %s --oio-account %s'
+                % (meta2, opts, self.account))
+            self.assertIn(fullname, output.split('\n'))
+
+        output = self.openio('container delete %s --oio-account %s'
+                             % (container, self.account))
+        for meta2 in meta2s:
+            if meta2.endswith(','):
+                meta2 = meta2[:-1]
+            output = self.openio_admin(
+                'meta2 list containers %s %s --oio-account %s'
+                % (meta2, opts, self.account))
+            self.assertNotIn(fullname, output.split('\n'))
+
+    def test_rawx_list_containers(self):
+        opts = self.get_format_opts(fields=['Status'])
+        container = random_str(16)
+        obj = random_str(16)
+        with tempfile.NamedTemporaryFile() as f:
+            f.write('test')
+            f.flush()
+            output = self.openio(
+                'object create %s %s --name %s %s --oio-account %s'
+                % (container, f.name, obj, opts, self.account))
```

Comment:
Since we don't want to test `openio object {create,locate,delete}` and to avoid slowing the test, we should use `ObjectStorageApi.{object_create,object_locate,object_delete}`.